Krivorotova Tatyana/style.css

```diff
@@ -2,100 +2,108 @@ body{
     margin: 0;
 }
 
-#widget{
+.weather-widget{
+    width: 612px;
+    height: 408px;
+    margin: 0;
     display: flex;
     flex-wrap: wrap;
     justify-content: center;
     flex-direction: row;
-    width: 612px;
-    height: 408px;
     font-family: 'Lato', sans-serif;
     color: #FFFFFF;
-    margin: 0;
+    
 }
 
-.elem{
+.weather-widget__city-card{
+    width: 204px;
+    height: 204px;
     display: flex;
     flex-direction: column;
     justify-content: center;
     align-items: center;
-    width: 204px;
-    height: 204px;
-    background-color: #00C0E2;
 }
 
-#a{
+.city-card-chicago{
     background-color: #00C0E2;
 }
 
-
-#b{
+.city-card-seattle{
     background-color: #5AD998;
 }
 
-#c{
+.city-card-brooklyn{
     background-color: #7558F8;
 }
 
-#d{
+.city-card-portland{
     background-color: #E9C14D;
 }
 
-#e{
+.city-card-schedule{
     justify-content: space-between;
     background-color: #E55679;
 }
 
-#f{
+.city-card-sanjose{
     background-color: #CB70D6;
 }
 
-#g{
-    height: 102px;
-    width: 204px;
-    background-color: #CB4C6C;
+.city-card-schedule__img{
+    margin-top: 25px;
+    margin-right: 10px;
 }
 
-#i span{
-    font-size: 11px;
+.city-card-schedule__schedule-table{
+    height: 99px;
+    width: 204px;
+    background-color: #CB4C6C;
+    box-shadow: inset 0 0 30px rgba(0,0,0,0.15);
 }
 
-#i{
-    width: 163px;
-    height: 70px;
-    margin-left: 20px;
-    margin-top: 15px;
+.city-card-schedule__week-days-container{
+    width: 160px;
+    height: 67px;
+    margin-left: 23px;
+    margin-top: 19px;
     display: flex;
     flex-direction: column;
-    justify-content: space-around;
+    justify-content: space-evenly;
 }
 
-#e img{
-    margin-top: 20px;
+.city-card-schedule__days-list{
+    display: flex;
+    justify-content: space-between;
+    list-style: none;
+    width: 155px;
+    height: 68px;
+    padding-inline-start: 0;
+    margin-block-start: 0;
+    margin-block-end: 0;
 }
```

Comment:
The .city-card-schedule__days-list element's three last styles have to be changed in order to be properly displayed on IE/Edge

Reply:
you can use `padding: 0;` and `margin: 0;` for this purposes. It should overwrite `padding-inline-start` and `margin-block-start`, `margin-block-end` in modern browsers and will work for old

Reply:
It works! Thank you